src/components/ResumeSubscriptionPopup/ResumeSubscriptionPopup.js

```diff
@@ -0,0 +1,249 @@
+import Loader from 'components/Loader';
+import checkmarkIcon from 'assets/images/checkmarkBase';
+import { ReactComponent as Close } from 'assets/images/errors/close.svg';
+import eventDispatcher, {
+  MSSDK_SWITCH_POPUP_ACTION_SUCCESSFUL,
+  MSSDK_SWITCH_POPUP_ACTION_FAILED
+} from 'util/eventDispatcher';
+
+import {
+  ContentStyled,
+  TitleStyled,
+  TextStyled,
+  ButtonWrapperStyled
+} from 'components/InnerPopupWrapper/InnerPopupWrapperStyled';
+import SkeletonWrapper from 'components/SkeletonWrapper';
+import {
+  ImageWrapper,
+  ArrowStyled,
+  SubscriptionIconStyled,
+  ImageStyled
+} from './ResumeSubscriptionPopupStyled';
+
+const ResumeSubscriptionPopup = ({
+  toOffer,
+  fromOffer,
+  hideInnerPopup,
+  updateList,
+  isPopupLoading
+}) => {
+  const STEPS = {
+    RESUME_DETAILS: 'RESUME_DETAILS',
+    CONFIRMATION: 'CONFIRMATION'
+  };
+
+  const STEPS_NUMBERS = {
+    RESUME_DETAILS: 1,
+    CONFIRMATION: 2
+  };
+
+  const { t } = useTranslation();
+  const [step, setStep] = useState(STEPS.RESUME_DETAILS);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setError] = useState(false);
+
+  const resumeSubscription = async () => {
+    setIsLoading(true);
+    try {
+      const resp = await subscriptionSwitch(
+        fromOffer.offerId,
+        toOffer.toOfferId,
+        toOffer.switchDirection
+      );
+      if (!resp.errors.length) {
+        eventDispatcher(MSSDK_SWITCH_POPUP_ACTION_SUCCESSFUL, {
+          fromOfferId: fromOffer.offerId,
+          toOfferId: toOffer.toOfferId,
+          switchDirection: toOffer.switchDirection,
+          algorithm: toOffer.algorithm
+        });
+        setIsLoading(false);
+        setStep(STEPS.CONFIRMATION);
+      } else {
+        eventDispatcher(MSSDK_SWITCH_POPUP_ACTION_FAILED, {
+          reason: resp.errors[0]
+        });
+        setError(true);
+        setIsLoading(false);
+      }
+    } catch {
+      eventDispatcher(MSSDK_SWITCH_POPUP_ACTION_FAILED);
+      setError(true);
+      setIsLoading(false);
+    }
+  };
+
+  const closePopupAndRefresh = () => {
+    hideInnerPopup();
+    updateList();
+  };
+
+  if (isPopupLoading) {
+    return (
+      <InnerPopupWrapper
+        steps={2}
+        popupTitle={t('resume-subscription-popup.title', 'Resume plan')}
+        currentStep={1}
+      >
+        <SkeletonWrapper
+          showChildren={false}
+          height={200}
+          width={450}
+          margin="auto"
+        />
+      </InnerPopupWrapper>
+    );
+  }
+
+  if (isError) {
+    return (
+      <InnerPopupWrapper
+        steps={2}
+        popupTitle={t('resume-subscription-popup.title', 'Resume plan')}
+        currentStep={STEPS_NUMBERS[step]}
+      >
+        <>
+          <ContentStyled>
+            <ImageWrapper>
+              <Close />
+            </ImageWrapper>
+            <TitleStyled step={step}>
+              {t(
+                'resume-subscription-popup.error-header',
```

Comment:
as this popup is only for pause I would change the translations key to a unique one for pause. Otherwise, publishers accidentally can break resume subscription popup